products/sle/main.pm

```diff
@@ -870,7 +870,10 @@ elsif (get_var("VIRT_AUTOTEST")) {
     }
     if (get_var("VIRT_PRJ1_GUEST_INSTALL")) {
         load_virt_guest_install_tests;
-        load_virt_feature_tests if (!(get_var("GUEST_PATTERN") =~ /win/img) && is_x86_64 && !get_var("LTSS"));
+        if (is_x86_64 && !(get_var("GUEST_PATTERN") =~ /win/img) && !get_var("LTSS")) {
+            load_virt_feature_tests;
+            loadtest "virt_autotest/validate_system_health";
```

Comment:
You should add a conditional parameter for test module validate_system_health, so we can turn it off if necessary.

Reply:
No problem. And https://gitlab.suse.de/qa-testsuites/openqa-job-group-yaml/-/merge_requests/138 is submitted.